lib/ecco/row_converter.rb

```diff
@@ -0,0 +1,61 @@
+module Ecco
+  module RubyConverter
+    FIRST_COLUMN = 0
+    SECOND_COLUMN = 1
+    KEY = 0
+    VALUE = 1
+    ROW = 0
```

Comment:
perhaps the name can be more descriptive here, like what data is found in the "first column" and so on

Reply:
Sure

Reply:
Fixed here: d3821566dc513a7d40e88c805a50f45202091f19